### PR TITLE
handle empty bid lists

### PR DESCRIPTION
**Pull request by morkusporkus**

### CODE REVIEW

File: src/Server/Features/OfferMatching/List.cs [hunk 1]
```diff
@@ -85,7 +85,7 @@ public record MatchPlayerCommand(int PlayerId) : IRequest { }
 public class MatchPlayerHandler : IRequestHandler<MatchPlayerCommand>
 {
     private readonly ApplicationDbContext _dbContext;
-
+    private const int _minimumValue = 1;
```
Comment (benjaminsampica) [inline src/Server/Features/OfferMatching/List.cs:88]: Put this inside of Bud

File: src/Tests/Features/OfferMatching/ListTests.cs [hunk 2]
```diff
@@ -91,6 +91,33 @@ public async Task GivenAnyAuthenticatedUser_WhenPlayerIsMatched_ThenPlayerIsMove
         result.YearAcquired.Should().Be(DateTime.Today.Year);
         result.ContractValue.Should().Be(int.MaxValue);
     }
+
+    [Fact]
+    public async Task GivenAnyAuthenticatedUser_WhenPlayerHasNoBids_ThenPlayerIsMovedToUnsignedStatus()
```
Comment (benjaminsampica) [inline src/Tests/Features/OfferMatching/ListTests.cs:96]: I’d drop all the asserts except the minimum bid and change this to “then has a contract value of one” or something?

File: src/Tests/Features/OfferMatching/ListTests.cs [hunk 3]
```diff
@@ -91,6 +91,29 @@ public async Task GivenAnyAuthenticatedUser_WhenPlayerIsMatched_ThenPlayerIsMove
         result.YearAcquired.Should().Be(DateTime.Today.Year);
         result.ContractValue.Should().Be(int.MaxValue);
     }
+
+    [Fact]
+    public async Task GivenAnyAuthenticatedUser_WhenPlayerHasNoBids_ThenContractValueIsOne()
+    {
+        int minimumBid = 1;
```
[the remark this answers is not in the document]
Comment (morkusporkus) [reply src/Tests/Features/OfferMatching/ListTests.cs:98]: Throw in a suggestion and I'll commit it in the car

**Comment by benjaminsampica** [comment]
Closes #99

**Review by benjaminsampica — APPROVED** [review]

**Pull request merged**